### PR TITLE
LF: Drop outdated TODOs

### PR DESCRIPTION
**Pull request by remyhaemmerle-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala [hunk 1]
```diff
@@ -151,13 +151,8 @@ object Error {
         with InternalError
 
     final case class DamlException(error: interpretation.Error) extends Error {
-      // TODO https://github.com/digital-asset/daml/issues/9974
-      //  For now we try to preserve the exact same message (for the ledger API)
-      //  Review once all the errors are properly structured
```
Comment (remyhaemmerle-da) [inline daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala:156]: KV Team will tackle that in Q4.

File: ledger/sandbox-classic/BUILD.bazel [hunk 2]
```diff
@@ -411,9 +411,6 @@ server_conformance_test(
 
 server_conformance_test(
     name = "conformance-test-exceptions",
```
Comment (cocreature) [inline ledger/sandbox-classic/BUILD.bazel:413]: I think we can just drop the whole suite? Exceptions are enabled by default.

File: ledger/sandbox-classic/BUILD.bazel [hunk 3]
```diff
@@ -427,9 +424,6 @@ server_conformance_test(
 
 server_conformance_test(
     name = "conformance-test-exceptions-append-only",
-    # TODO https://github.com/digital-asset/daml/issues/9914
```
Comment (cocreature) [inline ledger/sandbox-classic/BUILD.bazel:430]: same here

**Review by cocreature — APPROVED** [review]
lgtm with a minor comment on the exception tests

File: ledger/sandbox-classic/BUILD.bazel (hunk 2 above)
Comment (remyhaemmerle-da) [reply ledger/sandbox-classic/BUILD.bazel:413]: Good catch.

Comment (remyhaemmerle-da) [reply ledger/sandbox-classic/BUILD.bazel]: fb4fc905d976543c3b09d5599ae47000c5bb4b12

**Pull request merged**